mmv1/products/dlp/DiscoveryConfig.yaml

```diff
@@ -0,0 +1,352 @@
+                            name: 'undefined'
+                            description: |
+                              This field only has a name and description because of MM
+                              limitations. It should not appear in downstreams.
+                            values:
+                              - TABLE_MODIFIED_TIMESTAMP
+                        - !ruby/object:Api::Type::Enum
+                          name: 'frequency'
+                          description: 'How frequently data profiles can be updated when tables are modified. Defaults to never.'
+                          values:
+                            - :UPDATE_FREQUENCY_NEVER
+                            - :UPDATE_FREQUENCY_DAILY
+                            - :UPDATE_FREQUENCY_MONTHLY
+                - !ruby/object:Api::Type::NestedObject
+                  name: disabled
+                  description: 'Tables that match this filter will not have profiles created.'
+                  properties:
+                    []  # Meant to be an empty object with no properties - see here : https://cloud.google.com/sensitive-data-protection/docs/reference/rest/v2/organizations.locations.discoveryConfigs#disabled
+                    # The fields below are necessary to include the "disabled" filter in the payload
+                  send_empty_value: true
+                  allow_empty_object: true
+      - !ruby/object:Api::Type::Array
+        name: 'errors'
+        description: Output only. A stream of errors encountered when the config was activated. Repeated errors may result in the config automatically being paused. Output only field. Will return the last 100 errors. Whenever the config is modified this list will be cleared.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::NestedObject
+              name: details
+              description: Detailed error codes and messages.
+              properties:
+                - !ruby/object:Api::Type::Integer
+                  name: code
+                  description: The status code, which should be an enum value of google.rpc.Code.
+                - !ruby/object:Api::Type::String
+                  name: message
+                  description: A developer-facing error message, which should be in English. Any user-facing error message should be localized and sent in the google.rpc.Status.details field, or localized by the client.
+                - !ruby/object:Api::Type::Array
+                  name: 'details'
+                  description: A list of messages that carry the error details.
+                  item_type: Api::Type::KeyValuePairs
+            - !ruby/object:Api::Type::String
+              name: timestamp
+              description: The times the error occurred. List includes the oldest timestamp and the last 9 timestamps.
+      - !ruby/object:Api::Type::String
+        name: createTime
```

Comment:
These output only fields should be marked with `output: true`. We also usually omit create and update time fields from resources.

Reply:
Marked; I admit I just followed prior convention with existing DLP resources. I can remove the fields if that's preferred.